test/support/sdk/TestEnvironment.ts

```diff
@@ -253,14 +255,18 @@ export class TestEnvironment {
   }
 
   static stubNotification(config: TestEnvironmentConfig) {
-    global.window.Notification = global.Notification = {
+    global.Notification = {
```

Comment:
Was this the final thing that got `Notification` to show up in the service worker tests?

Reply:
yep. Iryna helped me out with this one